test/e2e/e2e_test.go

```diff
@@ -246,3 +323,25 @@ func sleepTestJobSet(ns *corev1.Namespace) *testing.JobSetWrapper {
 			Replicas(int32(replicas)).
 			Obj())
 }
+
+func shortSleepTestJobSet(ns *corev1.Namespace) *testing.JobSetWrapper {
```

Comment:
I don't see the point of two functions that do sleep with different times.

Reply:
Done, I parametrized the helper function with duration. I keep 20s for the old test, but we could consider reducing the time too. I tested that it passes anyway.